infra/main.bicep

```diff
@@ -72,6 +73,9 @@ param openAiSkuName string = 'S0'
 param openAiApiKey string = ''
 param openAiApiOrganization string = ''
 
+param huggingFaceApiKey string = ''
+param huggingFaceApiURL string = ''
+
```

Comment:
We're not using these here, are we?
```suggestion
```

Reply:
I feel like we should be passing these to the App Service configuration instead, if we expect the backend to be able to retrieve them from the environment. I don't think this works as is.